### PR TITLE
Initial commit for TextAnalysis notebook

### PR DESCRIPTION
**Pull request by dlukyanyuk**
Notebook for next analysis (most common words, frequencies)
Used languages: python, js 
Used libraries: d3 cloud component, matplotlib python library

### CODE REVIEW

File: TextAnalysis/README.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Words statistic representation of any txt file. This example uses d3 tag cloud component and python matplotlib library.
```
Comment (scottdraves) [inline TextAnalysis/README.txt]: rename to README.md and use markdown formatting.

**Comment by scottdraves** [comment]
put text like this in a markdown cell.  just put the heading here.

screen shot 2015-08-07 at 9 49 48 am

**Comment by scottdraves** [comment]
what's this for:
```
if __name__ == "__main__":
```

**Comment by scottdraves** [comment]
```
  textLocation = "C:\Users\beakertesting\.beaker\TagCloud.txt"
  deletionsLocation = "C:\Users\beakertesting\.beaker\deletions.txt"
  replaceDictionaryLocation = "C:\Users\beakertesting\.beaker\replaces.txt"
```
At the top of the notebook, define a single variable with a path used as the prefix for all the files.

**Comment by dlukyanyuk** [comment]
Should the path variable be defined at the top of whole notebook or just in a code cell?

**Comment by scottdraves** [comment]
use a code cell at the top of the notebook.

**Comment by dlukyanyuk** [comment]
Done.

**Comment by scottdraves** [comment]
```
textLocation = resourcesPath + "\TagCloud.txt"
```
backslash only works on windows.   use forward slash to be portable to all systems.

**Comment by scottdraves** [comment]
screen shot 2015-08-11 at 6 12 45 pm

error from last cell

**Comment by scottdraves** [comment]
break up the main python cell into 9 cells, one for the imports, 1 for each def, and 1 for the action at the end.

**Comment by scottdraves** [comment]
testing comment here.

**Comment by scottdraves** [comment]
word 
screen shot 2015-08-12 at 1 16 58 pm
cloud has too big a margin

**Comment by scottdraves** [comment]
remove unused style block
screen shot 2015-08-12 at 1 19 52 pm

**Comment by scottdraves** [comment]
https://github.com/twosigma/beaker-notebook-examples/pull/2#issuecomment-130098241
this applies to this notebook as well.

**Comment by scottdraves** [comment]
remove all the try/except statements, they are just distracting

**Pull request merged**